zeroex/orderwatch/order_watcher.go

```diff
@@ -1730,6 +1730,14 @@ func (w *Watcher) addAssetDataAddressToEventDecoder(assetData []byte) error {
 		}
 		w.eventDecoder.AddKnownERC1155(decodedAssetData.Address)
 		w.contractAddressToSeenCount[decodedAssetData.Address] = w.contractAddressToSeenCount[decodedAssetData.Address] + 1
+	case "StaticCall":
+		var decodedAssetData zeroex.StaticCallAssetData
+		err := w.assetDataDecoder.Decode(assetData, &decodedAssetData)
+		if err != nil {
+			return err
+		}
+		// NOTE(jalextowle): Only one staticcall is supported right now,
+		// so we don't neeed to add any addresses to the event decoder
```

Comment:
I think this comment should be adjusted to explain that the only currently supported static call is to a function that does not rely on on-chain state and therefore no additional contract events need to be watched to watch those orders.